test/runner/lib/core_ci.py

```diff
@@ -188,6 +196,25 @@ def _get_parallels_endpoints(self):
 
         raise ApplicationError('Unable to get available endpoints.')
 
+    def _get_vmware_endpoints(self):
```

Comment:
Supporting multiple endpoints doesn't add value unless thresholds are implemented by the ansible-core-ci service.

Reply:
Since we don't have vcenter/clustering deployed/licensed in worldstream atm, we would need to scale by adding additional standalone esxis, thus I figured we'd handle them as individual endpoints.  Is this not the case, or just that you prefer to wait until we actually have multiple hypervisors to worry about it?

Reply:
I figure we wait until we have multiple endpoints before implementing this. Depending on how long it is before that happens, the way we implement this may change.

Reply:
Gotcha, thanks for clarifying.